clio-util/src/main/scala/org/broadinstitute/clio/util/model/UpsertId.scala

```diff
@@ -72,6 +79,11 @@ object UpsertId {
     */
   private val next6bitIndex = (_: Int) => source.nextInt(Encoding.length)
 
+  /**
+    * Clio name for an upsert ID.
+    */
+  val UpsertIdFieldName: String = "upsertId"
```

Comment:
I think this'd make more sense to live next to the corresponding field for `entity_id` in `ElasticsearchIndex`, since it's really an artifact of how we've decided to keep notes in our indexes and not an inherent property of the `UpsertId` type.

Reply:
Makes sense, I can move this and `EntityId` into ElasticsearchIndex and get rid of the camel case names.